ui/hooks/featureflags.ts

```diff
@@ -5,5 +5,8 @@ export type FeatureFlags = { [key: string]: string };
 
 export function useFeatureFlags() {
   const { featureFlags } = useContext(CoreClientContext);
-  return { data: { flags: featureFlags || {} } };
+
+  const isFlagEnabled = (flag: string) => featureFlags?.[flag] === "true";
+
+  return { isFlagEnabled, flags: featureFlags };
```

Comment:
```suggestion
  return { isFlagEnabled, flags: featureFlags || {} };
```

To preserve old behaviour? I'm not sure what the loading pattern here looks like, if we might get undefined until flags are loaded etc?